src/registry/RegistryClient.ts

```diff
@@ -3,6 +3,7 @@ import { LogFunction } from '../utils';
 
 export type RegistryClientOptions = {
   log?: LogFunction;
+  useHttps?: boolean;
```

Comment:
Also: Since this is an implementation-specific option, I don't think it makes sense to put it in the interface's options type (because it won't apply to all implementations). Since it is only implemented in the `FHIRRegistryClient`, I think you should probably create a `FHIRRegistryClientOptions` type for this.

Reply:
Good point, I was so focused on the module, I forgot to step back and think about the name. Updated to use a `FHIRRegistryClientOptions` in [73ee511](https://github.com/FHIR/fhir-package-loader/pull/52/commits/73ee5112c4370cb04b68c431590a0a4f99cd83fb).